src/interfaces/database/ScheduledEvent.ts

```diff
@@ -0,0 +1,19 @@
+import { Document } from "mongoose";
+
+export interface RawScheduledEvent {
+  member: string;
+  time: number;
+  targetChannel: string;
+  lang: string;
+  message: string;
+}
+
+export type ScheduledEvent = RawScheduledEvent & Document;
```

Comment:
```suggestion
export interface ScheduledEvent extends Document {
  member: string;
  time: number;
  targetChannel: string;
  lang: string;
  message: string;
}
```

We should follow the style the other schemas use.